crates/plugins/tests/README.md

```diff
@@ -0,0 +1,8 @@
+## Fake plugin manifests
```

Comment:
Do we want to create a test or integration test for these, whether in this PR or a future one? Or is this good to keep around mainly for manual testing?

Reply:
I wasn't sure how to fit this stuff into the existing integration tests (not helped by my general woes with running the integration tests I admit).  I'd like to use them in a set of more unit-ish tests like the ones in the templates manager but not sure how - that might be a good thing to pick up in a later PR.  I would still say keep them to save maintenance programmers having to recreate them next time round!

Reply:
Sounds good to me!